### ISSUE TITLE
Automated release not working correctly

### ISSUE THREAD

**Issue by janivo** [inovex/elements#110]
### Describe the bug
When we try to release a new version, we use the lerna release command. Up to the last step - the release of NPM.js - everything runs correctly, but the release of the Angular Package fails without giving a helpful error message. 
 
### To Reproduce
⚠️  **Caution! When executing the lerna publish command, the core and the react packet will be irreversibly published.**

1. Clone the project
2. Install dependencies
3. Run `lerna publish --conventional-commits --no-changelog`
3. See the error on angular package

### Expected behavior
The angular package to be successfully released as well.

### Your Environment

| Software | Version |
| -------- | ------- |
| @inovex.de/elements | 1.0.2 |

**Comment by janivo** [comment]
I've just tested it again and here is a more detailed description of what's happening and what's the error:

1. Run `lerna publish --conventional-commits` 
> `--conventional-commits` in order to follow the semantic versioning
2. The packages are build and tested
3. Packages `elements` and `elements-react` publish successfully
4. Error on package `elements-angular`
```sh
lerna http fetch PUT 403 https://registry.npmjs.org/@inovex.de%2felements-angular 849ms
lerna ERR! E403 You cannot publish over the previously published versions: 1.0.2.
```
The reason is, that the build inside the `elements-angular/dist` is outdated with the old `package.json` where the old version is stated. In this case, the `version` field in `package.json` was `1.0.2` (the current version) but should be `1.1.0`. Even though the version is bumped correctly as you can see in the following output:
```
➜ lerna publish --conventional-commits
lerna notice cli v3.22.0
lerna info current version 1.0.2
lerna info Looking for changed packages since v1.0.2
lerna info getChangelogConfig Successfully resolved preset "conventional-changelog-angular"

Changes:
 - @inovex.de/elements-angular-builder: 1.0.2 => 1.1.0 (private)
 - @inovex.de/elements-react: 1.0.2 => 1.1.0
 - @inovex.de/elements: 1.0.2 => 1.1.0
 - @inovex.de/elements-storybook: 1.0.2 => 1.1.0 (private)
 - @inovex.de/elements-angular: 1.0.2 => 1.1.0
```
5. To fix this, I ran `yarn build` inside of `elements-angular` and published the package by hand with `npm publish dist/`.


So in order to fix this issue, we have to find the reason why the build files are outdated even though the bumping takes place before the build command.

**Comment by janivo** [comment]
Closed with #168
